src/language/language.rs

```diff
@@ -305,6 +313,9 @@ pub enum RelayOperator {
     /// (relay-operator relay-relu <data: access>)
     RelayReLU,
 
+    /// (relay-operator relay-relu <data: access> <data: access>)
```

Comment:
```suggestion
    /// (relay-operator relay-relu <data: access> <some-other-name: access>)
```
The parameters have identical names

Reply:
oops this is supposed to be alpha!